src/com/chocohead/mm/Plugin.java

```diff
@@ -270,8 +268,8 @@ public boolean removeAll(Collection<?> c) {
 	}
 
 	static byte[] makeMixinBlob(String name, Collection<? extends String> targets) {
-		ClassWriter cw = new ClassWriter(0);
-		cw.visit(52, Opcodes.ACC_PUBLIC + Opcodes.ACC_ABSTRACT + Opcodes.ACC_INTERFACE, name, null, "java/lang/Object", null);
+		ClassWriter cw = new ClassWriter(ClassWriter.COMPUTE_FRAMES); // Not sure whether you want this flag due to the slight performance overhead but not having to rewrite frames ourselves would be lovely :D
```

Comment:
There is no benefit to computing the frames for classes that have no methods since there's no frames to be computed ;)

Not like frames are especially hard to do yourself for small changes in general (more sweeping changes are another story), but for this writer especially this change isn't needed

Reply:
Ah perhaps I identified the wrong writer then
My bad